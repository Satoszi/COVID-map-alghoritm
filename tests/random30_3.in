30 30 15 125
random30_3
+.....+..+++++|.+|+..+.+++..-.
.-..+.+.+.++.+++.--.+-|.++.++-
+-.+.+++++.+.+.+...+++-++..+.|
...+...|--+|..-.+-.+.+.+++.+-.
.+.+.-...-+|.-++|.+..+|.+++-.+
.+++++|..--+..+|++++.+.+..-.++
+....-.+++|.+.+-++.+...+-+++|.
...|..-++-.++|+...-|+.+++|+-|+
+..++.-|..+.--++++-+..+++.-+.+
+.+.|+++++||+...+-++-...|+.|+-
+.+|++|.-|.+|.++...--+-..-+-.+
...+|......+|.||+..+++.+.+.+++
+++.|-+.+.-..-.|-+|..|.||++.++
+...++++-..+.|.+.|-..+++..-..+
+.+++++|+|.+++..+|-+++..-..++.
|+-..++..+.+.|+..+++|+-.++|.+|
+|+.+...+-+++.+.+.+.+++++-.+++
++-.+...-.|+...+.|++..+.+.|+.+
++||-.+|..+|.....+-..++-+.+++|
....+....+.++++.|++|+.|....|++
+-++++.|.+|.++.-.--.-++.-.-+.+
...+.|+.+||.++||+|+.+++.+.+++.
+.+..--||.|...+..+-.....++|+-.
+-+-+++||++..+...+.+...+++-..-
+..+|.++...-...+...|.+.|.++-++
|.-+|.+-+|-++++.+++++-.++.-+.+
+.++.-.-+.-+--.+++-.+..+.|+...
+.+-....+....-.|.+..+++..-.+..
++-+...+++|+|..++.+++.-|...++.
++.|.-+++..+.+..++.-++++.++.+.
