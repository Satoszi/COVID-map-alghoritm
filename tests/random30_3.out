25 29
17 29
13 29
3 29
22 28
18 28
12 28
11 28
10 28
27 27
25 27
13 27
26 26
25 26
5 26
26 25
10 25
9 25
21 24
15 24
11 24
3 24
29 23
13 23
7 22
5 22
24 21
18 21
17 21
16 21
15 21
14 21
8 21
6 21
24 20
17 20
15 20
10 20
7 20
20 19
19 19
16 19
9 19
4 19
7 18
6 18
4 18
3 18
2 18
17 17
15 17
8 17
16 16
4 16
2 16
26 15
25 15
14 15
27 14
24 14
16 14
9 14
8 14
26 13
18 13
11 13
24 12
18 12
16 12
13 12
10 12
8 12
5 12
0 12
25 11
16 11
29 10
25 10
14 10
8 10
3 10
28 9
25 9
12 9
11 9
2 9
26 8
10 8
6 8
0 8
29 7
20 7
18 7
13 7
12 6
10 6
9 6
5 6
0 6
26 5
23 5
6 5
29 4
22 4
5 4
16 3
14 3
8 3
29 2
3 2
0 2
27 1
22 1
17 1
4 1
1 1
28 0
23 0
21 0
18 0
17 0
16 0
9 0
6 0
0 0
